lib/routers/establishment/projects.js

```diff
@@ -207,6 +216,12 @@ router.put('/:id/update-licence-holder',
   submit('update')
 );
 
+router.put('/:id/revoke',
```

Comment:
can I be a major pedant and request we add the check here as a middleware? Couple of reasons, firstly because all other validation checks happen in the middleware chain, and secondly we will bail a sooner if it's not able to be actioned.

```
(req, res, next) => {
  if (req.project.status !== 'active') {
    return next(new BadRequestError());
  }
  next();
},
perms(blah)
```

Reply:
we've got a 
```
case 'delete':
          if (req.project.status !== 'inactive') {
            throw new BadRequestError('Active projects cannot be deleted.');
          }

          return req.workflow.delete({
            ...params,
            id: req.project.id
          });
```
but ok if it makes you happy.